samples/matter/smoke_co_alarm/Kconfig

```diff
@@ -5,6 +5,16 @@
 #
 mainmenu "Matter nRF Connect Smoke CO Alarm Example Application"
 
+config SYSTEM_OFF
+	bool "Enable system off on idle"
+	depends on POWEROFF
+	depends on SOC_NRF54L15_CPUAPP
```

Comment:
Do we want to support this on L05 or L10?

Reply:
For L05, definitely not, but maybe for L10. I will change it to `SOC_SERIES_NRF54LX`